src/prefect/tasks/core/operators.py

```diff
@@ -32,6 +32,27 @@ def run(self, task_result: Any, key: Any) -> Any:  # type: ignore
         return task_result[key]
 
 
+class GetAttr(Task):
+    """
+    Helper task that retrieves a specific attribute of an upstream task's result.
+
+    Args:
+        - *args (Any): positional arguments for the `Task` class
+        - **kwargs (Any): keyword arguments for the `Task` class
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+
+    def run(self, task_result: Any, item: Any) -> Any:  # type: ignore
+        """
+        Args:
+            - task_result (Any): a value
+            - item (Any): the attribute to retrieve as `task_result.attr`
```

Comment:
Would be good to also note that nested parameter access also works (e.g. `attr="a.b.c.d"`).